src/domain/Reward.ts

```diff
@@ -125,12 +143,12 @@ export function calculateRewards(date: BigInt, pool: Pool): void {
   systemRewards.save()
 }
 
-function checkRewardRate(checker: RewardDayTotal): void {
-  if (checker.toDateRewardAggregateValue.gt(BigDecimal.fromString(tierOneRewards))) {
-    log.debug('pending rewards > 1 MILL:  {}', [checker.toDateRewardAggregateValue.toString()])
+function setRewardRate(systemRewards: RewardDayTotal): RewardDayTotal {
+  if (systemRewards.toDateRewardAggregateValue.lt(BigDecimal.fromString(tierOneRewards))) {
```

Comment:
No need to update if it's already correct:

```suggestion
  if (systemRewards.toDateRewardAggregateValue.lt(BigDecimal.fromString(tierOneRewards)) && systemRewards.rewardRate.neq(BigDecimal.fromString('0.0042')) {
```